### ISSUE TITLE
ClientMapEvictAllTest.evictAll_firesOnlyOneEvent

### ISSUE THREAD

**Issue by hasancelik** [hazelcast/hazelcast#6654]
```
java.lang.AssertionError: null
    at org.junit.Assert.fail(Assert.java:86)
    at org.junit.Assert.assertTrue(Assert.java:41)
    at org.junit.Assert.assertFalse(Assert.java:64)
    at org.junit.Assert.assertFalse(Assert.java:74)
    at com.hazelcast.client.map.ClientMapEvictAllTest.evictAll_firesOnlyOneEvent(ClientMapEvictAllTest.java:82)
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.x-IbmJDK1.8/com.hazelcast$hazelcast-client/21/testReport/junit/com.hazelcast.client.map/ClientMapEvictAllTest/evictAll_firesOnlyOneEvent/

**Comment by cangencer** [comment]
@sancar looks like event is being fired twice

**Comment by sancar** [comment]
fixed by #6728

**Comment by hasancelik** [comment]
```
java.lang.AssertionError: expected:<1> but was:<2>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.client.map.ClientMapEvictAllTest.evictAll_firesOnlyOneEvent(ClientMapEvictAllTest.java:83)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-ZuluJDK8/com.hazelcast$hazelcast-client/706/testReport/junit/com.hazelcast.client.map/ClientMapEvictAllTest/evictAll_firesOnlyOneEvent/
